airflow/www/templates/airflow/trigger.html

```diff
@@ -117,7 +117,7 @@
       </textarea>
     {% elif form_details.schema and ("integer" in form_details.schema.type or "number" in form_details.schema.type) %}
       <input class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}"
-             {% if "integer" in form_details.schema.type %} valuetype="number" type="number" {%else%} valuetype="decimal" type="decimal" {% endif %}
+             valuetype="number" {% if "integer" in form_details.schema.type %} type="number" {%else%} type="decimal" {% endif %}
```

Comment:
```suggestion
             valuetype="number" {% if "integer" in form_details.schema.type %} type="number" {% else %} type="decimal" {% endif %}
```

Good change to fix this formatting